src/templates/account/index.html

```diff
@@ -0,0 +1,285 @@
+{% extends 'base/main.html' %}
+
+{% block title %} Task {% endblock %}
+
+{% block content %}
+
+{% load static %}
+
+<script src="
+    https://cdn.jsdelivr.net/npm/jsgantt-improved@2.8.10/dist/jsgantt.min.js
+    "></script>
+    <link href="
+    https://cdn.jsdelivr.net/npm/jsgantt-improved@2.8.10/dist/jsgantt.min.css
+    " rel="stylesheet">
+
+
+    <div style="position:relative" class="gantt" id="GanttChartDIV"></div>
+
+    
+    <script>
+
+        var g = new JSGantt.GanttChart(document.getElementById('GanttChartDIV'), 'day');
+    
+        g.setOptions({
+            vCaptionType: 'Complete',  // Set to Show Caption : None,Caption,Resource,Duration,Complete,     
+            vQuarterColWidth: 36,
+            vDateTaskDisplayFormat: 'day dd month yyyy', // Shown in tool tip box
+            vDayMajorDateDisplayFormat: 'mon yyyy - Week ww',// Set format to dates in the "Major" header of the "Day" view
+            vWeekMinorDateDisplayFormat: 'dd mon', // Set format to display dates in the "Minor" header of the "Week" view
+            vLang: 'en',
+            vShowTaskInfoLink: 1, // Show link in tool tip (0/1)
+            vShowEndWeekDate: 0,  // Show/Hide the date for the last day of the week in header for daily
+            vAdditionalHeaders: { // Add data columns to your table
+                category: {
+                    title: 'Category'
+                },
+                sector: {
+                    title: 'Sector'
+                }
+            },
+            vUseSingleCell: 10000, // Set the threshold cell per table row (Helps performance for large data.
+            vFormatArr: ['Day', 'Week', 'Month', 'Quarter'], // Even with setUseSingleCell using Hour format on such a large chart can cause issues in some browsers,
+    
+        });
+    
+        let arr = [
```

Comment:
let's remove the test data, if this is already using the Gantt data API

Reply:
deleted the file as we don't need this one. Currently we are using the ```src/templates/chart/main.html``` for job-task gantt-chart and dashboard view